tgui-next/packages/tgui/interfaces/LaborClaimConsole.js

```diff
@@ -0,0 +1,67 @@
+import { toTitleCase } from 'common/string';
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Box, Button, LabeledList, Section, Table } from '../components';
+
+export const LaborClaimConsole = props => {
+  const { act, data } = useBackend(props);
+  const {
+    can_go_home,
+    id_points,
+    ores,
+    status_info,
+    unclaimed_points,
+  } = data;
+  return (
+    <Fragment>
+      <Section>
+        <LabeledList>
+          <LabeledList.Item label="Status">
+            {status_info}
+          </LabeledList.Item>
+          <LabeledList.Item label="Shuttle controls">
+            <Button
+              content="Move shuttle"
+              disabled={!can_go_home}
+              onClick={() => act('move_shuttle')} />
+          </LabeledList.Item>
+          <LabeledList.Item label="Points">
+            {id_points}
+          </LabeledList.Item>
+          <LabeledList.Item label="Unclaimed points">
+            {unclaimed_points}
+            <Button
+              ml={1}
+              content="Claim"
+              disabled={!unclaimed_points}
+              onClick={() => act('claim_points')} />
```

Comment:
put this button in the `buttons` prop on the labeledlist item

Reply:
Done.